backend/src/main/java/com/woowacourse/moragora/entity/Participant.java

```diff
@@ -47,4 +47,8 @@ public void mapMeeting(final Meeting meeting) {
             meeting.getParticipants().add(this);
         }
     }
+
+    public boolean isUserMaster(final Long id) {
```

Comment:
이 메서드가 적절한가에 대한 의문점이 조금 드네요.
어떤 특정한 Participant에게 '니가 이 User이고 Master야?' 라는걸 묻는걸로 보여요.
두 가지 일을 한다고 보여지는데  어떻게 생각하시나요?

그리고 만약 이 메서드를 유지한다면 파라미터를 좀 더 명시적으로 하면 좋을 거 같아요! 메서드명 또한 조금 더 명시적이면 좋을거 같습니당.

```suggestion
    public boolean isUserMaster(final Long userid) {
```

Reply:
저는 parameter에 `User` 객체를 넘겨주고 `isMaster`라는 메서드명도 좋아보여요!